.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: Test Refactor
+on:
+  push:
+    branches-ignore:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Test
+        run: doit test
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Do we need the creds to run the tests?

Reply:
In fact it could run without it, but the task that check credentials runs on almost every task. Let me remove them and check.

Reply:
we don't need creds. it failed usually in Travis as well (this step), but it doesn't stop the pipeline.

Reply:
(I also run this regularly without AWS creds locally)

Reply:
I removed the credentials and `creds` task references in all doit task other than `login` and it worked. I'll keep like that.